server/webserver.go

```diff
@@ -912,6 +912,16 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 				return
 			}
 
+			for _, o := range orchestrators {
+				dbO, err := s.LivepeerNode.Database.SelectOrchs(&lpcommon.DBOrchFilter{
```

Comment:
Might be worth adding a non-zero length check for the returned slice just to be safe. The length should usually be non-zero, but I think there could be an edge case where this endpoint is hit after a new address is added to the on-chain pool, but before the address is added to the DB by the orchestrator watcher.

Reply:
good point, if `len(dbO) == 0` we should set o.PricePerPixel to `big.NewRat(0, 1)` otherwise the floatstring conversion in the CLI will fail.

Reply:
good point, if `len(dbO) == 0` we should set o.PricePerPixel to `big.NewRat(0, 1)` otherwise the floatstring conversion in the CLI will fail. 

Fixed in eac1d3b9